### PR TITLE
fix: Prisma schema and API update

### PR DESCRIPTION
**Pull request by ruheni**
Updated `schema.prisma` file and rest-express API example to match GraphQL examples

### CODE REVIEW

File: typescript/rest-express/src/index.ts [hunk 1]
```diff
@@ -1,16 +1,25 @@
-import { PrismaClient } from '@prisma/client'
-import * as bodyParser from 'body-parser'
```
Comment (nikolasburk) [inline typescript/rest-express/src/index.ts:2]: nice! just a little reminder if we replace `bodyParser` here we can also remove the dep from `package.json`

File: typescript/rest-express/src/index.ts [hunk 2]
```diff
@@ -48,6 +84,27 @@ app.delete(`/post/:id`, async (req, res) => {
...
+})
+
+app.get('/user/:id/drafts', async (req, res) => {
+  const { id } = req.params
+  const { email } = req.body
+
+  const drafts = await prisma.user.findUnique({
+    where: {
+      id: Number(id) || undefined,
+      email: email || undefined
```
Comment (nikolasburk) [inline typescript/rest-express/src/index.ts]: can you double check whether the `|| undefined` is actually needed here? I believe this was specific to Nexus so we _should_ be able to omit it in non-Nexus project. if it doesn't produce any compile-time or runtime errors, we can leave it out.

Comment (ruheni) [reply typescript/rest-express/src/index.ts]: This isn't needed. Thank you for spotting it. 🙂

File: typescript/rest-hapi/src/plugins/posts.ts [hunk 3]
```diff
@@ -66,12 +66,30 @@ export default usersPlugin
...
+    const posts = await prisma.post.findMany({
+      where: {
+        published: true,
+        ...or
+      },
       include: { author: true },
+      take: Number(take) || undefined,
+      skip: Number(skip) || undefined,
+      orderBy: {
+        updatedAt: orderBy || undefined
```
Comment (nikolasburk) [inline typescript/rest-hapi/src/plugins/posts.ts:88]: are these `|| undefined`s necessary here?

Comment (ruheni) [reply typescript/rest-hapi/src/plugins/posts.ts:88]: Yes. If not added, it defaults to `NaN`.
Here's the error that is thrown if ` || undefined` is omitted: 
```bash
PrismaClientValidationError:
Invalid `prisma.post.findMany()` invocation:

{
  where: {
    published: true
  },
  include: {
    author: true
  },
  take: NaN,
        ~~~
  skip: NaN,
        ~~~
  orderBy: {
    updatedAt: 'desc'
  }
}
```

**Comment by nikolasburk** [comment]
Great job Alex!!! 🚀

**Pull request merged**